geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/GeoMesaConfigurator.scala

```diff
@@ -43,6 +43,7 @@ object GeoMesaConfigurator {
   private val sftKey           = s"$prefix.sft"
   private val transformsKey    = s"$prefix.transforms.schema"
   private val transformNameKey = s"$prefix.transforms.name"
+  private val desiredSplits = s"$prefix.splitCount.userPreference"
```

Comment:
private val desiredSplits = s"$prefix.splitCount.userPreference"
maybe should be 
 private val desiredSplits = s"$prefix.mapreduce.split.count.hint"
or some such.